shell/platform/fuchsia/flutter/accessibility_bridge_unittest.cc

```diff
@@ -127,6 +127,42 @@ TEST_F(AccessibilityBridgeTest, DeletesChildrenTransitively) {
   EXPECT_FALSE(semantics_manager_.UpdateOverflowed());
 }
 
+TEST_F(AccessibilityBridgeTest, PopulatesStates) {
+  flutter::SemanticsNode node0;
+  node0.id = 0u;
```

Comment:
nit: flutter semantics nodes are not unsigned ints - fuchsia semantics nodes are.